bin/installer.sh

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
this file is what we run in order to create our binary :)

You have to run it in different operating system to get the different binaries (run it in windows to get a .exe, run it in linux to get a .bin, etc...)